portfolio/src/test/java/com/google/sps/NameTest.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletOutputStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public final class NameTest extends Mockito {
+  private final DataServlet servlet = new DataServlet();
+
+  @Mock
+  private HttpServletRequest request;
+  @Mock
+  private HttpServletResponse response;
+  @Mock
+  private ServletConfig servletConfig;
+  @Mock
+  private ServletOutputStream outputStream;
+
+  @Test
+  public void getMyName() throws IOException {
+   when(servlet.getServletConfig()).thenReturn(servletConfig);
```

Comment:
In order to use when(...).thenReturn(...), you need to initialize servletConfig and outputStream when you declare them as class variables above. This is probably why you are getting the Mockito error.